components/brave_ads/core/internal/account/user_data/README.md

```diff
@@ -1,45 +1,54 @@
 # User Data
 
-**Optional** user data may not be included in the confirmation payload.
+Return to [confirmation redemption](../../account/utility/redeem_confirmation/README.md).
 
-**Mutable** user data can change when a confirmation redemption fails and is retried later.
+> [!IMPORTANT]
+> Approval from the privacy team is required for changes to existing user data or for new user data.
 
-## User Data Submitted when Redeeming Anonymous Confirmations
+## Non-Brave Rewards User
 
-Included in the confirmation payload for non-Brave Rewards and Brave Rewards users. See [anonymous confirmation redemption](../utility/redeem_confirmation/README.md).
+### User Data Submitted when Redeeming Anonymous Confirmations
 
-| user data  | optional  | mutable  | description  |
-|---|---|---|---|
-| conversion  | yes  | no  | Conversion action and encrypted verifiable envelope.  |
-| httpResponseStatus  | yes  | no  | Whether the navigation committed an error page.  |
+Included as part of the "confirmation" payload. See [non-Brave Rewards user confirmation payload](../confirmations/non_reward/README.md).
 
-### Additional User Data Submitted for Brave Rewards Users
+| key  | example  | description  |
+|---|---|---|
+| conversion.action[^2]  | {"conversion":[{"action":"click"}]}  | Click-through conversion attribution to determine which advertising efforts led a user to click on an ad and subsequently complete a desired action.  |
```

Comment:
Click-through conversions is the only possibility for Search Ads, but there's more to include here for Brave News, right?

Reply:
Great catch. I will update. Thank you.

Reply:
Added. Thanks